discovery-infra/start_discovery.py

```diff
@@ -394,7 +408,7 @@ def nodes_flow(client, cluster_name, cluster):
         cluster_info = client.cluster_get(cluster.id)
         macs = utils.get_libvirt_nodes_macs(nodes_details["libvirt_network_name"])
 
-        if not (cluster_info.api_vip and cluster_info.ingress_vip) and args.master_count != 1:
+        if not (cluster_info.api_vip and cluster_info.ingress_vip):
```

Comment:
not sure i understand this condition, maybe you know?

Reply:
Whoever did it didn't want to set vips or machine-cidr for none platform